.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+## Description
+
+Add a brief summary of your PR here, between one sentence and two paragraphs
+
+## Additions
+
+- Detail API additions here if any
+- Do not list changes or removals
+
+## Removals
+
+- List API removals here if any
+
+## Modifications
+
+- List changes to existing structures and functions here if any
+
+## PR Checklist
+
+By placing an x in the boxes I certify that I have
```

Comment:
For punctuation consistency, can you add a `:` here, and a fullstop at the end of every bullet point?